File: FindWeather/WeatherData.java
import org.json.JSONObject;

public class WeatherData {
    private String cityName;
    private double latitude;
    private double longitude;
    private double temperature;
    private String weatherCondition;
    private String dataSource;
    private String fetchedTime;

    public WeatherData(String cityName, double lattitude, double longitude, double temperature, String weatherCondition, String dataSource){
        this.cityName = cityName;
        this.latitude = lattitude;
        this. longitude = longitude;
        this.temperature = temperature;
        this.weatherCondition = weatherCondition;
        this.dataSource = dataSource;
    }

    public String getCityName(){
        return  cityName;
    }
    public String getTime(){
        return fetchedTime;
    }
    public double getLatitude() {
        return latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public double getTemperature() {
        return temperature;
    }
    public String getWeatherCondition(){
        return weatherCondition;
    }
    public String getDataSource(){
        return dataSource;
    }

    

File: FindWeather/210042111_Lab08/WeatherData.cs
namespace _210042111_Lab08
{  public class WeatherData
    {       public string CityName { get; set; }
        public string WeatherCondition { get; set; }
        public double WeatherTemperature { get; set; }
        public string LocationInformation { get; set; }

        public WeatherData(string CityName, string WeatherCondition, double WeatherTemperature)
        {
            this.CityName = CityName;
            this.WeatherCondition = WeatherCondition;
            this.WeatherTemperature = WeatherTemperature;
        }

    }
}
